RecoTracker/LSTCore/src/alpaka/PixelTriplet.h

```diff
@@ -2666,12 +2726,14 @@ namespace SDL {
 
       //special dispensation to tilted PS modules!
       if (moduleType == 0 and moduleSubdet == SDL::Barrel and moduleSide != Center) {
-        error /= alpaka::math::sqrt(acc, 1.f + drdz * drdz);
+        float drdz = modulesInGPU.drdzs[lowerModuleIndex];
+        error /= alpaka::math::sqrt(acc, 1 + drdz * drdz);
       }
       RMSE += (residual * residual) / (error * error);
     }
 
-    RMSE = alpaka::math::sqrt(acc, 0.2f * RMSE);
+    RMSE = alpaka::math::sqrt(acc, 0.2f * RMSE);  //the constant doesn't really matter....
```

Comment:
This number is used to form the normalized chi2 distribution. In the square root, we need to divide the sum by the degree of freedom. We have residual of 5 MDs, so in the square root it divides by 5 here. I agree that this constant does not affect our performance in principal. If we also enlarge the threshold number by 5 times, we can get the same results. I think it's a bit helpful to keep the 0.2 factor, so this function is mathematically a "chi2" distribution so that we can imagine the shape.

Reply:
Then I would write the explanation about the degrees of freedom being 5, instead of saying that it doesn't really matter.